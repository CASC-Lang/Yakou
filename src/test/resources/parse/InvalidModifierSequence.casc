mut priv class InvalidModifierSequence

impl InvalidModifierSequence {
    mut ovrd fn a1(self) {}

    ovrd priv fn a2() {}
}