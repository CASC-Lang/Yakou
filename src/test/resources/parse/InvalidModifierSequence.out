error: Cannot declare access modifier after `mut` keyword
--> ./InvalidModifierSequence.casc:1:0
1 | mut priv class InvalidModifierSequence
    ^^^= hint: `mut` keyword here
2 |
error: Cannot declare `ovrd` keyword after `mut` keyword
--> ./InvalidModifierSequence.casc:4:4
3 | impl InvalidModifierSequence {
4 |     mut ovrd fn a1(self) {}
        ^^^= hint: `mut` keyword here
5 |
error: Cannot declare access modifier after `ovrd` keyword
--> ./InvalidModifierSequence.casc:6:4
5 |
6 |     ovrd priv fn a2() {}
        ^^^^= hint: `ovrd` keyword here
7 | }